.github/workflows/release.yml

```diff
@@ -10,54 +10,46 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@master
+
       - uses: actions/setup-node@v2
         with:
           node-version: '12.22.5'
           cache: 'yarn'
+
       - name: Git Unshallow
         run: git fetch --prune --unshallow
+
       - name: Fetch Node Packages
         run: |
           yarn --frozen-lockfile
-      - name: Get Previous Tag
-        id: previous_tag
-        uses: 'WyriHaximus/github-action-get-previous-tag@v1'
-      - name: Get Current Version
-        id: extract_version
-        uses: Saionaro/extract-package-version@v1.0.6
-      - name: Validate Semver
-        id: validate_semver
-        run: |
-          valid_latest_tag=$(node ./node_modules/semver/bin/semver.js ${{ steps.previous_tag.outputs.tag }})
-          valid_current_version=$(node ./node_modules/semver/bin/semver.js ${{ steps.extract_version.outputs.version }})
-          change_log=$(node -e "require('./scripts/changelog.js').build({ writeToFile: true }).then(cl => console.log(cl))")
 
-          # if not change log is truncated at first line
-          change_log="${change_log//'%'/'%25'}"
-          change_log="${change_log//$'\n'/'%0A'}"
-          change_log="${change_log//$'\r'/'%0D'}"
```

Comment:
Why was this changed?

Reply:
This code was copied here, in the studio and in messaging. I created a single action which extracts all the information we need and made the release process simpler. You can check the information here:

https://github.com/botpress/gh-actions/blob/master/get_release_details/src/index.ts